tests/testgraphics.c

```diff
@@ -179,48 +180,39 @@ static void test_hdc ()
 	status = GdipGetImageGraphicsContext (image, &graphics);
 	assert(status == Ok);
 
-// This causes libgdiplus to crash.
-#if 0
 	status = GdipGetDC (NULL, &hdc);
 	assert (status == InvalidParameter);
 
 	status = GdipGetDC (graphics, NULL);
 	assert (status == InvalidParameter);
-#endif
 
 	status = GdipGetDC (graphics, &hdc);
 	assert (status == Ok);
 	assert(hdc != 0);
 
-// Libgdiplus does not yet match this GDI+ behaviour.
-#if 0
 	status = GdipGetDC (graphics, &hdc);
 	assert (status == ObjectBusy);
-#endif
 
 	status = GdipReleaseDC (NULL, hdc);
 	assert (status == InvalidParameter);
 
-// This causes libgdiplus to crash.
-#if 0
 	status = GdipReleaseDC (NULL, 0);
 	assert (status == InvalidParameter);
-#endif
 
 	status = GdipReleaseDC (graphics, 0);
 	assert (status == InvalidParameter);
 
 	status = GdipReleaseDC (graphics, hdc);
 	assert (status == Ok);
 
-// Libgdiplus does not validate that the graphics has an hdc before releasing.
-#if 0
 	status = GdipReleaseDC (graphics, hdc);
 	assert (status == InvalidParameter);	
-#endif
 
 	GdipDisposeImage (image);
 	GdipDeleteGraphics (graphics);
+
+	status = GdipGetDC (graphics, &hdc);
```

Comment:
This is one example, it's just pure luck that this still works in CI (it started randomly failing on my machine) since the memory could be overwritten at any time.